code/fastinfoset/pom.xml

```diff
@@ -25,46 +25,29 @@
         <version>1.3.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
+
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.sun.xml.fastinfoset</groupId>
     <artifactId>FastInfoset</artifactId>
     <packaging>jar</packaging>
     <version>1.3.0-SNAPSHOT</version>
     <name>fastinfoset</name>
-    <url>http://fi.java.net</url>
+
     <build>
-        <defaultGoal>install</defaultGoal>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-            </resource>
-        </resources>
+
         <testResources>
             <testResource>
                 <directory>src/test/resources</directory>
             </testResource>
         </testResources>
+
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
                     <archive>
                         <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-                        <manifest>
-                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
-                        </manifest>                        
-                        <manifestEntries>
-                            <Specification-Title>ITU-T Rec. X.891 | ISO/IEC 24824-1 (Fast Infoset)</Specification-Title>
-                            <Specification-Version>1.0</Specification-Version>
-                            <Implementation-Title>Fast Infoset Implementation</Implementation-Title>
-                            <Implementation-Version>${project.version}</Implementation-Version>
-                            <Implementation-Build-Id>${scmBranch}-${buildNumber}, ${timestamp}</Implementation-Build-Id>
-                            <Implementation-Vendor>Oracle</Implementation-Vendor>
-                            <Implementation-Vendor-Id>com.oracle</Implementation-Vendor-Id>
-                            <Extension-Name>com.sun.xml.fastinfoset</Extension-Name>
-                            <URL>${project.url}</URL>
-                        </manifestEntries>
                     </archive>
                     <mavenExecutorId>forked-path</mavenExecutorId>
                     <useReleaseProfile>false</useReleaseProfile>
```

Comment:
I'd remove mavenExecutorId, useReleaseProfile and arguments from here probably

Reply:
OK, they are gone.